scripts/test/Muon/muon_data_context_test.py

```diff
@@ -88,6 +88,20 @@ def test_dead_time_table_returns_correctly(self):
     def test_return_sample_log_returns_correctly(self):
         self.assertEqual(self.context.get_sample_log('goodfrm').value, 31369.0)
 
+    def test_that_is_multi_period_returns_false_for_single_period_data(self):
+        is_multi_period = self.context.is_multi_period()
+        
+        self.assertTrue(not is_multi_period)
+
+    def test_that_any_multi_period_data_will_mark_everything_as_multiperiod(self):
+        load_result = {'OutputWorkspace': [1,2,3,4]}
+        self.loaded_data.add_data(workspace=load_result, run=[84447], filename='workspace_filename', instrument='EMU')
+        self.context._current_runs = [[84447],[19489]]
```

Comment:
how many periods do each of these runs have? It is not obvious to me

Reply:
We are not actually loading a multiperiod workspace here but instead creating a fake instance of one via having an 'OutputWorkspace' member which has a length of greater than one. As this is not very clear currently I have updated the tests to populate this with the correct WorspaceWrapper objects rather than just numbers.